runner/src/main.rs

```diff
@@ -291,48 +291,63 @@ fn build_wasm_module(name: &str, target: &Target, example_name: &str) -> Step {
 
 fn build_server(opt: &BuildServer) -> Step {
     match opt.server_variant.as_str() {
-        "base" | "logless" => Step::Single {
-            name: format!("build server ({})", opt.server_variant),
-            command: Cmd::new_with_env(
-                "cargo",
-                spread![
-                    ...match &opt.server_rust_toolchain {
-                        // This overrides the toolchain used by `rustup` to invoke the actual
-                        // `cargo` binary.
-                        // See https://github.com/rust-lang/rustup#toolchain-override-shorthand
-                        Some(server_rust_toolchain) => vec![format!("+{}", server_rust_toolchain)],
-                        None => vec![],
-                    },
-                    "build".to_string(),
-                    // If building in coverage mode, use the default target from the host, and build
-                    // in debug mode.
-                    ...if opt.coverage {
-                        vec![]
-                    } else {
-                        vec![
-                            format!("--target={}", opt.server_rust_target.as_deref().unwrap_or(DEFAULT_SERVER_RUST_TARGET)),
-                            "--release".to_string(),
-                        ]
-                    },
-                    "--manifest-path=oak/server/rust/oak_loader/Cargo.toml".to_string(),
-                    ...if opt.server_variant == "logless" {
-                        vec!["--no-default-features".to_string()]
-                    } else {
-                        vec![]
-                    },
-                ],
-                &if opt.coverage {
-                    hashmap! {
-                        // Build the Runtime server in coverage mode, as per https://github.com/mozilla/grcov
-                        "CARGO_INCREMENTAL".to_string() => "0".to_string(),
-                        "RUSTDOCFLAGS".to_string() => "-Cpanic=abort".to_string(),
-                        // grcov instructions suggest also including `-Cpanic=abort` in RUSTFLAGS, but this causes our build.rs scripts to fail.
-                        "RUSTFLAGS".to_string() => "-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off -Zpanic-abort_tests".to_string(),
-                    }
-                } else {
-                    hashmap! {}
+        "base" | "logless" => Step::Multiple {
+            name: "server".to_string(),
+            steps: vec![
+                Step::Single {
+                    name: "create bin folder".to_string(),
+                    command: Cmd::new(
+                        "mkdir",
+                        vec!["--parent".to_string(), "oak/server/bin".to_string()],
```

Comment:
I think @daviddrysdale mentioned that `--parents` does not work on macos, only `-p` does, so maybe we should stick to that. Also BTW it is `--parents`, not `--parent`, so are you sure this actually worked? :)

Reply:
Done